### PR TITLE
Release version 0.25.1

### PR DESCRIPTION
**Pull request by ogoffart**
I hope i'm doing that right.

Looking forward to disable the wayland feature in my programs :-)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+# Version 0.25.0 (2020-10-10)
```
Comment (kchibisov) [inline CHANGELOG.md:3]:
```suggestion
# Version 0.25.1 (2020-10-10)
```

**Review by kchibisov — CHANGES_REQUESTED** [review]

**Comment by kchibisov** [comment]
>Looking forward to disable the wayland feature in my programs :-)

I wonder why you don't want to support Wayland, seems like a 'ok' platform to support, that has pretty good support in winit.

File: CHANGELOG.md (hunk 1 above)
Comment (ogoffart) [reply CHANGELOG.md]: oops :-s

**Comment by ogoffart** [comment]
> I wonder why you don't want to support Wayland,

Targeting an embedded platform using xorg.

**Review by kchibisov — APPROVED** [review]

**Pull request merged**